Tasks/AppCenterDistributeV1/package.json

```diff
@@ -17,8 +17,11 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks",
   "dependencies": {
+    "abort-controller": "^3.0.0",
+    "appcenter-file-upload-client-node": "1.0.0",
     "azure-storage": "^2.10.3",
     "jszip": "^3.1.2",
+    "node-fetch": "^2.6.0",
```

Comment:
It should be resolved as a dependency of `appcenter-file-upload-client-node` implicitly